python/flatbuffers/builder.py

```diff
@@ -104,6 +104,12 @@ def __init__(self, initialSize):
         self.objectEnd = None
         self.vtables = []
 
+    def MaxBufferSize(self):
```

Comment:
Please make this a constant on the class, so that we do not incur a function call each time.